### DISCUSSION TITLE
Does this protocol support the forum usecase?

### DISCUSSION THREAD

**Discussion by shi-yan** [bluesky-social/atproto#251] [How to do it?]
the document says data are saved in repositories. A repository is a user-signed storage that contains all data from a user.

But what about a forum? because forums are not user-oriented. A thread, once posted, should not belong to a specific user, hence its data shouldn't be saved into a user's repository.

Can this protocol support that use case? Can data migration still work for this use case? Or this protocol can only be used for microblogs/blogs?

**Comment by pfrazee** [comment]
This is an accurate reading of the base technology, but we've got some ideas for how this can be solved. We definitely plan to support the forum usecase. We'll do some writeups on the solution once it's stable.
